src/shared/components/post/post.tsx

```diff
@@ -1072,6 +1072,19 @@ export class Post extends Component<any, PostState> {
     });
   }
 
+  // No need to set finished on a comment vote, save, etc
+  findAndUpdateComment(res: RequestState<CommentResponse>) {
+    this.setState(s => {
+      if (s.commentsRes.state === "success" && res.state === "success") {
+        s.commentsRes.data.comments = editComment(
+          res.data.comment_view,
+          s.commentsRes.data.comments,
+        );
+      }
```

Comment:
The main was on `L1069` with `s.finished.set` 

It should only "finish" the comment (IE clear the reply / edit box), on either an edit or a comment create. It shouldn't close the box on a vote, save, etc. 

So I just had to create `findAndUpdateCommentEdit` alternatives that included the finished setting, and remove the finished on the existing ones.